airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/metadata/commands.py

```diff
@@ -24,6 +23,8 @@ def deploy(ctx: click.Context) -> None:
 @deploy.command(cls=DaggerPipelineCommand, name="orchestrator", help="Deploy the metadata service orchestrator to production")
 @click.pass_context
 async def deploy_orchestrator(ctx: click.Context) -> None:
+    # Import locally to speed up CLI.
+    from pipelines.airbyte_ci.metadata.pipeline import run_metadata_orchestrator_deploy_pipeline
```

Comment:
This shaves 4 seconds off of the execution time for `airbyte-ci --help`, bringing it down to under a second.

Reply:
Do you have a clue why the import is so slow?

Reply:
It's not this import in particular, it's all of the transitive imports in `pipeline.py` and beyond.